src/Pipelines/analyze_longread.php

```diff
@@ -382,71 +387,96 @@
 //variant calling
 if (in_array("vc", $steps))
 {
-	//determine basecall model
-	$basecall_model = get_basecall_model($used_bam_or_cram);
-	$basecall_model_path = "";
-
-	if ($basecall_model == "")
+	if ($sys['name_short'] == "LR-PB-SPRQ")
```

Comment:
Please add a function in genomics.php that determines if it is ONT or PB.
For now matching the name is ok, but add a TODO comment that we should come up with a better way

Reply:
I wasn't happy with the name-matching myself...

Put it in a separate function now, as requested.